SCons/Tool/packaging/__init__.py

```diff
@@ -96,17 +95,13 @@ def Package(env, target=None, source=None, **kw):
         raise UserError("No source for Package() given")
 
     # decide which types of packages shall be built. Can be defined through
-    # four mechanisms: command line argument, keyword argument,
-    # environment argument and default selection (zip or tar.gz) in that
-    # order.
-    try:
-        kw['PACKAGETYPE'] = env['PACKAGETYPE']
-    except KeyError:
-        pass
-
-    if not kw.get('PACKAGETYPE'):
+    # four mechanisms: command line argument, keyword argument, environment
+    # argument and default selection (zip or tar.gz) in that order.
+    kw.setdefault('PACKAGETYPE', env.get('PACKAGETYPE'))
+    if kw['PACKAGETYPE'] is None:
```

Comment:
Looks like this if statement is duplicated twice in a row?

Reply:
I'll take a look, yeah at a glance something extra seems to be left in.

Reply:
Yes, pushing update now.